src/examples/build-without-updating.yml

```diff
@@ -0,0 +1,15 @@
+description: >
```

Comment:
flip this example to be new example of adding the additional feature instead of disabling it.